src/vm/appdomain.cpp

```diff
@@ -6436,6 +6445,9 @@ HRESULT RuntimeInvokeHostAssemblyResolver(INT_PTR pManagedAssemblyLoadContextToB
             // If we couldn't resolve the assembly using TPA LoadContext as well, then
             // attempt to resolve it using the Resolving event.
             // Finally, setup arguments for invocation
+            tracer.Trace(BinderTracing::ResolutionAttemptedOperation::Stage::AppDomainAssemblyResolveEvent,
```

Comment:
`AppDomainAssemblyResolveEvent` -> `AssemblyLoadContextResolvingEvent`

Reply:
The `AppDomain.AssemblyResolve` event actually gets fired in `AssemblySpec::ResolveAssemblyFile` - called by `AppDomain::BindAssemblySpec` (via many other intermediary functions...). As we've discussed, I don't think we need to do all the stages and tests in one change; I think we can do that one in a follow-up.

Reply:
The extension points should be covered by @elinor-fung change right? Why do we need one more event around these? Or maybe I'm missing something.

Reply:
I thought of the extension point events I added as a bit more narrow (called this specific handler and that handler returned X), while the resolution 'stage' would also cover any additional validation the binder does for whether or not the assembly returned is acceptable (e.g. matching name, public key, collectible)

Reply:
That's a good point - makes sense. I just want to make sure we don't overdo it - too many events is not going to be helpful either 😄